Engine Hacks/_MasterHackInstaller.event

```diff
@@ -152,6 +152,13 @@ POP
   FreezeAllASMC:
   #incbin "ASM/freezeall.dmp"
 
+  // Chapter 27 ASMC-able functions
+  ALIGN 4
+  gChapter27FogLevel:
+    WORD Ch27FogLevel
+  
+  #include "CCBlitzChapter27Asmcs.lyn.event"
```

Comment:
Including the output of lyn rather than calling lyn directly?

Reply:
Given that the lyn *input* (the compiled object file) is already generated I feel like there'd be no gain in doing that (If we could include from source that'd be another story, but sadly here we do not do that).